.pre-commit-config.yaml

```diff
@@ -19,6 +19,8 @@ repos:
   rev: v2.4.0
   hooks:
   - id: codespell
+    additional_dependencies: ['tomli']
+    args: ['--toml', 'pyproject.toml']
     exclude: 'benchmarks/sonnet.txt|(build|tests/(lora/data|models/fixtures|prompts))/.*|vllm/third_party/.*'
```

Comment:
Can we delete this line now if `pre-commit` is getting this config from `pyproject.toml`?

Reply:
I tried that initially but it was failing here: https://github.com/vllm-project/vllm/actions/runs/13316648035/job/37192184214